src/app.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en" style="background-color: black !important">
+<html lang="en" class="bg-secondary">
```

Comment:
Please revert this, i had a long long discussion with @moonlitgrace over this

Reply:
@meowsocute he is right, this will cause [FOUC](https://en.wikipedia.org/wiki/Flash_of_unstyled_content)